test/openTerm.js

```diff
@@ -0,0 +1,35 @@
+const { existsSync, unlink } = require("fs");
+const { assert } = require("chai");
+const { after } = require("mocha");
+const log = require("electron-log");
+const openTermLinux = require("../app/electron/platform/linux/openTerm");
+const openTermWin = require("../app/electron/platform/windows/openTerm");
```

Comment:
As index of platform exports correct one, directly import that. In case that works incorrectly, then it will also be tested as well.